setup.py

```diff
@@ -97,6 +97,8 @@
         "waymo": [
             "waymo-open-dataset-tf-2-4-0",
             "tabulate==0.8.9",
+            "pathos==0.2.8",
+            "readline==6.2.4.1"
```

Comment:
Yes, `readline` is needed for tab completion,  to allow moving the cursor with arrow keys and scrolling through past commands during raw input.
`pathos` is needed for multiprocessing and parallelizing plot functions, since the default inbuild multiprocessing in python does not pickle functions.

Reply:
I see, I think we have `cloudpickle` for picking functions but it might be reasonable to use `pathos` isolated to the waymo dependencies. My only concern might be if they cause dependency issues with other core package options.